### PR TITLE
Feature/fleet pg

### PR DESCRIPTION
**Pull request by kcbreuil**

### CODE REVIEW

File: studio/schemas/documents/fleetCard.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+            title: 'Description',
+            type: 'text',
+        },
+        {
+            name: 'image',
+            title: 'Image',
+            type: 'mainImage',
+        },
+    ],
+};
```
Comment (asasmith) [inline studio/schemas/documents/fleetCard.js:22]: I don't think you'll need this file at all. You can get all of the data for the ui card element by querying for the individual fleet items.

File: studio/schemas/documents/fleetCardSection.js [hunk 2]
```diff
@@ -0,0 +1,13 @@
...
+    type: 'document',
+    fields: [
+        {
+            name: 'fleet',
+            title: 'Fleet',
+            type: 'array',
+            of: [{ type: 'reference', to: { type: 'fleetCard' } }],
+        },
+    ],
+};
```
Comment (asasmith) [inline studio/schemas/documents/fleetCardSection.js:13]: I don't think you'll need this either. All of this data can be queried at the index page level.

**Review by asasmith — CHANGES_REQUESTED** [review]

File: studio/schemas/documents/fleetCard.js (hunk 1 above)
Comment (kcbreuil) [reply studio/schemas/documents/fleetCard.js:22]: so should we include a description section on the individual fleet item for the card? @asasmith

Comment (asasmith) [reply studio/schemas/documents/fleetCard.js:22]: Yes. We can also reuse that field for seo description

**Review by asasmith — APPROVED** [review]

**Pull request merged**